maxim/logger/openai/async_responses.py

```
@@ -0,0 +1,198 @@
+    async def __aexit__(self, exc_type, exc, tb):
+        try:
+            get_final = getattr(self._stream, "get_final_response", None)
+            if get_final is not None:
+                try:
+                    result = get_final()
+                    if inspect.isawaitable(result):
+                        result = await result
+                    self._final_response = result
+                except Exception:
+                    # If calling get_final_response fails, skip capturing final response
+                    pass
```

Ruff (0.14.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)